lib/erb_lint/cache.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module ERBLint
+  class Cache
+    CACHE_DIRECTORY = ".erb-lint-cache"
+
+    def initialize(config, file_loader = nil)
+      @config = config
+      @file_loader = file_loader
+      @hits = []
+      @new_results = []
+      puts "Cache mode is on"
```

Comment:
Not sure how this is handled in the rest of the library but maybe we should pass in an io or logger so we could silence these outputs?

Something like

```ruby
def initialize(config, file_loader = nil, logger = STDOUT)
  # ...
  @logger = logger
  @logger.puts "Cache mode is on"
end
```